### ISSUE TITLE
GLPI web interface fields are not properly labeled

### ISSUE THREAD

**Issue by adrianorjales** [glpi-project/glpi#11122]
### Code of Conduct

- [ ] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [ ] I have searched the existing issues

### Version

GLPI 9.5.6

### Bug description

When navigating through all the web UI using a screen reader software like NVDA, the labels of the input fields are not spoken correctly because of the lack of labels on these fields
For example, when i move the focus to a text field to input the name of a computer, the screen reader software does not read any information about if it is a field to input the name or a description. It only says Edit text, instead of saying "Name, edit text". This information is gatered from the '<label for' html attribute on the page.

### Steps To reproduce

1. With any configuration, (default), 
2. Create an inventory item, like a computer
3. With NVDA installed and running, check that the name of any field when focused is not spoken.

**Comment by cconard96** [comment]
I don't have the ability to try NVDA currently since it is for Windows only, but I did check the Computer form using the WAVE WebAIM plugin for Chrome on GLPI 10, and it looks like the accessibility has improved a lot with this version with the new UI.

If you are able, can you test the latest GLPI v10 beta to see how well it works for you? I suggest not using it in production yet though since it isn't a final release.

**Comment by adrianorjales** [comment]
i will try it and tell you something.

**Comment by adrianorjales** [comment]
I have tested the rc3 glpi 10 version. 
The Accessibility is better, but most fields remain unlabeled, for example, the status of a computer on the computer details page, the type of computer, the technician that manages the hardware, the manufacturer, etc.

**Comment by cconard96** [comment]
Does it seem like mainly just the dropdowns that aren't labelled properly?

**Comment by adrianorjales** [comment]
I can't check all the fields on the UI, but most of the fields have missing label. Including those of settings tab

**Comment by trasher** [comment]
Many UI fields are still displayed with old methods, while some other are using the new UI with Twig. This is a very huge job to migrate the entire UI, and we're now focused on the 10.0 release. This will certainly improve on next releases.
